recipes/flatcc/all/FlatccGenerateSources.cmake

```diff
@@ -0,0 +1,88 @@
+
```

Comment:
Can you please point me to the cmake script from the flatbuffers repo at https://github.com/dvidelabs/flatcc
containing these functions/macros?

Reply:
I wrote that cmake script because it simplifies using flatcc in cmake a lot. I am going to send it upstream as well.
Do you want me to add it to the flatcc package via the patches mechanism?

Reply:
Edit: PR for flatcc cmake module submitted: https://github.com/dvidelabs/flatcc/pull/169

Reply:
Please do it as a patch, so when your pr gets accepted, this recipe can be adapted more easily.
In the mean time, maybe add a FIXME/TODO that this is experimental conan-only code?